File: web/src/lib/features.ts
/**
 * Produkt-Feature-Flags (Web).
 *
 * Europa ist vorerst aus — Code und Routen bleiben erhalten.
 * Wieder aktivieren:
 *   1) NEXT_PUBLIC_ENABLE_EUROPE=true  (z. B. in Vercel / .env.local), oder
 *   2) `europe: true` unten als Default setzen.
 */
export const features = {
  europe: process.env.NEXT_PUBLIC_ENABLE_EUROPE === "true",
} as const;

File: app/lib/features.py
"""Produkt-Feature-Flags (Streamlit).

Europa vorerst aus — Seite bleibt unter pages/, wird nur nicht verlinkt.
Wieder aktivieren: ENABLE_EUROPE=1 in der Umgebung oder europe=True unten.
"""

from __future__ import annotations

import os

features = {
    "europe": os.environ.get("ENABLE_EUROPE", "").strip().lower() in {"1", "true", "yes"},
}
